packages/query/src/stacks/token-metadata/non-fungible-tokens/non-fungible-token-holdings.query.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { useCurrentNetworkState } from '../../../leather-query-provider';
+import { QueryPrefixes } from '../../../query-prefixes';
+import { StacksClient, useStacksClient } from '../../stacks-client';
+
+const staleTime = 15 * 60 * 1000; // 15 min
+
+const queryOptions = { cacheTime: staleTime, staleTime, refetchhOnFocus: false } as const;
+
+function fetchNonFungibleTokenHoldings(client: StacksClient) {
+  return async (address: string) => {
+    if (!address) return;
+    return client.hiroNonFungibleTokensApi.getNftHoldings(address);
+  };
+}
+
+function makeNonFungibleTokenHoldingsQuery(address: string, network: string, client: StacksClient) {
```

Comment:
Should be exported.

Reply:
I am actually going to merge this one so I can remove the queries from the extension. I'm getting quite a few conflicts there from other pkg changes. I'll start refactoring the queries in a follow up PR.